spk/ffmpeg/src/service-setup.sh

```diff
@@ -5,10 +5,12 @@ service_postinst ()
 {
     # setuid for proper vaapi access
     chmod u+s /var/packages/ffmpeg/target/bin/ffmpeg
+    chmod u+s /var/packages/ffmpeg/target/bin/vainfo
 }
 
 service_postupgrade ()
 {
     # setuid for proper vaapi access
     chmod u+s /var/packages/ffmpeg/target/bin/ffmpeg
+    chmod u+s /var/packages/ffmpeg/target/bin/vainfo
 }
```

Comment:
According to https://help.synology.com/developer-guide/synology_package/scripts.html `service_postupgrade` may be discarded.

Reply:
But for now it still stands, right?

Reply:
Yes, it is harmless but also useless as postinst is called by postupgrade during upgrade sequence.